snuba/web/query.py

```diff
@@ -72,6 +73,7 @@ def _run_query_pipeline(
         )
         clickhouse_query = StorageProcessingStage().execute(clickhouse_query)
     else:
+        request_copy = copy.deepcopy(request)
```

Comment:
Can you change this so the copy only happens if `try_new_query_pipeline` is true?

Reply:
Good catch. Updated, but I had to wrap `EntityAndStoragePipelineStage` with two if-statements so it doesn't look pretty, but it's just temporary.